src/edu/csus/ecs/pc2/core/IInternalController.java

```diff
@@ -176,7 +178,13 @@ void submitJudgeRun(Problem problem, Language language, SerializedFile mainFile,
      * @param packet
      */
     void sendToTeams(Packet packet);
-
+    
+    /**
+     * Send to groups and or individual teams specified by ultimateDestination.
+     * @param packet
+     * @param ultimateDestination
```

Comment:
A comment indicating what this is would be helpful.  That is, what is "ultimateDestination" specifying?

Reply:
removed ultimateDestination, it now is done by two different arrays one for groups one for teams